06_gpu_and_ml/protein-folding/boltz_predict.py

```diff
@@ -90,10 +79,18 @@ def main(
 # Because Modal images include [GPU drivers](https://modal.com/docs/guide/cuda) by default,
 # installation of higher-level packages like `boltz` that require GPUs is painless.
 
+# Since protein folding requires an input yaml file and an input msa file, we add the "data" directory in our local directory to the image.
```

Comment:
1. They don't just need an input yaml file and an input msa file. They may need a number of files. We're simplifying this by saying "well, just make sure all the files are in the `data` directory".
2. I would split the explanation in two parts. First, separate the Image definition into two steps -- installing the package and then adding the data directory. Explain that we mount the local data directory so that it's accessible to the remote code. Elsewhere, when explaining how to use this script, note that the input YAML files and any files it references, e.g. MSA files, need to be in that directory.